package/Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM ubuntu:16.04
 RUN apt-get update && apt-get install -y kmod curl nfs-common fuse \
-        libibverbs1 librdmacm1 libconfig-general-perl libaio1
+        libibverbs1 librdmacm1 libconfig-general-perl libaio1 && rm -rf /var/lib/apt/lists/*
```

Comment:
Thanks @circa10a -- this is good. 
Can we make a small change to push this to the next line with "\"